modules/unsupported/process-feature/src/main/java/org/geotools/process/vector/PointStackerProcess.java

```diff
@@ -395,19 +573,34 @@ private SimpleFeatureType createType(CoordinateReferenceSystem crs, boolean stre
         tb.add(ATTR_GEOM, Point.class, crs);
         tb.add(ATTR_COUNT, Integer.class);
         tb.add(ATTR_COUNT_UNIQUE, Integer.class);
-        tb.add(ATTR_BOUNDING_BOX_GEOM, Polygon.class);
-        tb.add(ATTR_BOUNDING_BOX, String.class);
+        tb.add(ATTR_BOUNDING_AREA, Geometry.class);
+        tb.add(ATTR_BOUNDING_BOX, List.class);
+
         if (stretch) {
             tb.add(ATTR_NORM_COUNT, Double.class);
             tb.add(ATTR_NORM_COUNT_UNIQUE, Double.class);
         }
+        tb.add(ATTR_CLUSTERED_VALUES, JSONArray.class);
         tb.setName("stackedPoint");
         SimpleFeatureType sfType = tb.buildFeatureType();
         return sfType;
     }
 
     private static class StackedPoint {
+        /** First Element */
+        private Coordinate first = null;
+        /** Differential Coordinate X */
+        private double dx = 0;
+        /** Differential Coordinate Y */
+        private double dy = 0;
+
+        /** Cell Size */
+        private double cellSize = 0;
+
+        /** Internal Key */
         private Coordinate key;
+        /** Default Value, but it will be overridden at initialization (constructor) */
+        ClusterType clusterType = ClusterType.Natural;
```

Comment:
As I said in the commend is useless, when I wrote that code it was part of something else, but given the fact that I needed the feature fast I did not take enough time to look over those issues. It will be left uninitialized here as previous member variables.